client/src/components/MyProfile.js

```diff
@@ -240,15 +357,51 @@ function Home({ currentUser, setCurrentUser }) {
                 City={user.city}
                 Country={user.country}
                 Email={user.email}
-            />
-        ) : (
-            <p>user is false</p>
-        )}
+              />
+            )
+          }
+        </div>
+        {/* This is shwon when the user has selected "My Events" from the top menu. */}
+        <div className='MyEventsArea' ref={eventsRef}>
+          {loadingEvents ?
+            (
+              <p className='HintParagraphBig'>Loading...</p>
+            )
+            :
+            (
+              <div>
+                {events.length === 0 ?
+                  (
+                    <p className='HintParagraphBig' style={{margin: "30px 0 0 0", fontStyle: "italic"}}>You are not partisipating in any events</p>
+                  ) 
+                  :
+                  (
+                    <div style={{width: "100%", display: "flex",flexDirection: "column", alignItems: "center", overflow: "scroll", maxHeight: "65vh"}}>
+                      {events.map((event, index) => 
+                      (
+                        <EventItem
+                        // The admin abilities are always disabled meaning that the admin can only edit the events from the Home view.
```

Comment:
At least for me, the edit button was still visible, so either the comment or the functionality is wrong :d